src/Calculator.java

```diff
@@ -1,36 +1,38 @@
 
 public class Calculator {
 	public static void main(String[] args) {
-		int ra = Add(4, 2);
-		int rs = Sub(4, 2);
-		int rm = Multiply(4, 2);
-		int rd = Div(4, 2);
+		Calculator calculator = new Calculator();
+		
+		int ra = calculator.Add(4, 2);
+		int rs = calculator.Sub(4, 2);
+		int rm = calculator.Multiply(4, 2);
+		int rd = calculator.Div(4, 2);
 		System.out.println(ra + "\n" + rs + "\n" + rm + "\n" + rd);
 		int elements [] = {2,4,6,8,9,10};
-		System.out.println(calculateaverage(elements));
+		System.out.println(calculator.calculateaverage(elements));
 	}
 
-	public static int Add(int num1, int num2) {
+	public  int Add(int num1, int num2) {
 		int result = num1 + num2;
 		return result;
 	}
 
-	public static int Sub(int num1, int num2) {
+	public  int Sub(int num1, int num2) {
 		int result = num1 - num2;
 		return result;
 	}
 
-	public static int Multiply(int num1, int num2) {
+	public  int Multiply(int num1, int num2) {
 		int result = num1 * num2;
 		return result;
 	}
 
-	public static int Div(int num1, int num2) {
+	public  int Div(int num1, int num2) {
 		int result = num1 / num2;
 		return result;
 	}
 	
-	public static double calculateaverage (int [] elements)
+	public double calculateaverage (int [] elements)

```

Comment:
Follow camel case convention (i.e instead of `calculateaverage` use `calculateAverage` )